server/src/main/java/io/druid/query/lookup/MapLookupExtractorFactory.java

```diff
@@ -103,4 +119,29 @@ public int hashCode()
     result = 31 * result + (isOneToOne ? 1 : 0);
     return result;
   }
+
+  public static class MapLookupIntrospection implements LookupIntrospectHandler
```

Comment:
I see the some places we use "Introspect" while in other places we used "Introspection", can you make it consistent by putting "Introspection" in all classes.
nit: can we call it MapLookupIntrospectionHandler instead ?